lib-multisrc/gigaviewer/src/eu/kanade/tachiyomi/multisrc/gigaviewer/GigaViewer.kt

```diff
@@ -315,21 +371,19 @@ abstract class GigaViewer(
     }
 
     private fun String.toDate(): Long {
-        return runCatching { DATE_PARSER.parse(this)?.time }
+        return runCatching { (if (isPaginated) DATE_PARSER_COMPLEX else DATE_PARSER_SIMPLE).parse(this)?.time }
```

Comment:
This should use `DATE_PARSER_SIMPLE` only. Plus, remove this `toDate()` and use `DATE_PARSER_SIMPLE.tryParse()` at the call site.

https://github.com/keiyoushi/extensions-source/blob/98a4e1540993d487aa254e7626bb688d06731cda/core/src/main/kotlin/keiyoushi/utils/Date.kt#L7